test/language/block-scope/syntax/redeclaration-in-block/attempt-to-redeclare-let-binding-with-var.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) Copyright 2011 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: B.3.3
+description: >
+    redeclaration within block:
+    attempt to redeclare let binding with var
+negative: SyntaxError
+---*/
+{ let f; var f; }
```

Comment:
I think this test is wrong? The var decl is hoisted out so there should be no error afaict?